CHANGELOG.md

```diff
@@ -142,6 +142,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
     - simplified configuration of localized router
     - fixed directory path in `shopsys.domain_images_url_prefix`
     - using Elasticsearch as a datasource for Read model
+- [#68 - Universal logging to administration of public demoshop](https://github.com/shopsys/demoshop/pull/68)
```

Comment:
This change does not belong to the same commit with the rest of the changes (assuming you don't want to squash afterwards)

Reply:
Fixed